libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.spec.ts

```diff
@@ -606,9 +619,110 @@ describe('Date range picker', function () {
     expect(calculatorIdControl?.errors).toEqual(expectedError);
   }));
 
-  it('should be accessible', async () => {
```

Comment:
This "removed" test still exists but is just relabeled as `should be accessible with no label` within the new describe block.